lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -907,28 +907,6 @@ def test_financial_assistance_login_required(self):
             response = self.client.get(url)
             self.assertRedirects(response, reverse('signin_user') + '?next=' + url)
 
-    def test_bypass_course_info(self):
```

Comment:
It seems like overriding the flag would allow legacy tests to continue to work while the legacy functionality still exists.  Since we aren't using this functionality, these tests are really the only protection we have against breaking something that is still supported in the code.

Reply:
changed